src/randoop/reflection/VisibilityPredicate.java

```diff
@@ -0,0 +1,15 @@
+package randoop.reflection;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+public interface VisibilityPredicate {
```

Comment:
It's especially important to document anything that is an interface or abstract class or intended to be overridden.  I came here trying to understand some other code but this didn't explain my questions.